src/error.rs

```diff
@@ -68,20 +64,6 @@ impl Error {
         Error::authentication_error(mechanism_name, "invalid server response")
     }
 
-    /// Attempts to get the `std::io::Error` from this `Error`.
```

Comment:
This was only necessary for the custom trust-dns implementation, so it's not needed anymore